msgraph/administrative_units_test.go

```diff
@@ -177,6 +236,17 @@ func testAdministrativeUnitsClient_RemoveMembers(t *testing.T, c *test.Test, adm
 	}
 }
 
+func testAdministrativeUnitsClient_CreateGroup(t *testing.T, c *test.Test, administrativeUnitId string, g *msgraph.Group) (group *msgraph.Group) {
+	group, status, err := c.AdministrativeUnitsClient.CreateGroup(c.Context, administrativeUnitId, g)
+	if err != nil {
+		t.Fatalf("AdministrativeUnitsClient.CreateGroup(): %v", err)
+	}
+	if status < 200 || status >= 300 {
```

Comment:
We expect status code 201, so anything else should be an error, right?
```suggestion
	if status < 201 || status > 201 {
```

Reply:
rewritten:

```go
if status != http.StatusCreated {
```